examples/simple-file-server/server/udp_file_server.bal

```diff
@@ -36,14 +36,15 @@ isolated service on new udp:Listener(PORT) {
         lock {
             // Initialize the flag for "End of File Reached".
             boolean eofReached = false;
-            byte[] newData = [];
-            // Copy data to be edited.
-            newData = data.clone();
+            byte[]|error newData = data.cloneWithType();
+            if newData is error {
+                return error (newData.message());
```

Comment:
```suggestion
                return error udp:Error(newData.message());
```

Reply:
`return error (newData.message());` also does the same thing, right?